### DISCUSSION TITLE
Anyone else getting crashes and unable to move the taskbar on latest Win 11 update?

### DISCUSSION THREAD

**Discussion by sanjaysrikonda66** [valinet/ExplorerPatcher#1507] [General]
Win 11 update: 22H2: Build 23466.1001

None of the taskbar items are working and sometimes causing the taskbar to crash altogether. Anyone else getting this?

I'm using version 22621.1555.55.2 build of ExplorerPatcher.

Just curious if there's another update available that fixes this or is this the issue because of the latest Win 11 update?

**Comment by tallman69** [comment]
It is most likely the update..  ExplorerPatcher does not support Windows preview builds.  The latest build EP currently supports is 22621.1702.
For more information see this discussion on Symbols:  https://github.com/valinet/ExplorerPatcher/wiki/Symbols

**Reply by tallman69** [reply]
@sanjaysrikonda66 

Have you tried uninstalling the Windows preview update to see if that resolves the issues?  If that doesn't solve the problem, try uninstalling and reinstalling the latest stable build of ExplorerPatcher after the Windows preview update is uninstalled.

**Comment by PantheraTigris1** [comment]
It was crashing for me too and I am not on the preview build. I am still on 22621.1702 but EP crashes after every minor Windows update. 

The solution for me is to delete the registry for explorer patcher and restart the PC after every Windows update.

**Comment by sanjaysrikonda66** [comment]
I fixed it by deleting the registry and restarting the app. Great advice.
